src/libs/ReportUtils.ts

```diff
@@ -4208,6 +4223,38 @@ function buildOptimisticApprovedReportAction(amount: number, currency: string, e
     };
 }
 
+/**
+ * Builds an optimistic APPROVED report action with a randomly generated reportActionID.
+ */
+function buildOptimisticUnapprovedReportAction(amount: number, currency: string, expenseReportID: string): OptimisticUnapprovedReportAction {
```

Comment:
Maybe this should go in `ReportActionUtils`?

Reply:
@Beamanator `ReportUtils` is scary. It has too many methods which are for a `reportAction`

Reply:
Asked on slack just in case there's a deep meaning to all this, or just an oversight - https://expensify.slack.com/archives/C01GTK53T8Q/p1719365046871369

Reply:
Thanks for bringing that to slack, seeeeeems we're currently ok with keeping it here even though it looks like it belongs in ReportActionUtils :D Probs would require moving a LOT of functions around